### PR TITLE
Add arihanc.com testnet server

### PR DESCRIPTION
**Pull request by danny91**
Only just managed to get around to this, server has actually been up a few months.

### CODE REVIEW

File: lib/servers_testnet.json [hunk 1]
```diff
@@ -9,4 +9,8 @@
     "electrontest.cascharia.com": {
         "s": "51002"
     }
```
Comment (marceloneil) [inline lib/servers_testnet.json]: You're missing a comma here

File: lib/servers_testnet.json [hunk 2]
```diff
@@ -9,4 +9,8 @@
...
+    "bchtestnet.arihanc.com": {
+        "t":"53001",
+        "s":"53002"
+    },
```
Comment (marceloneil) [inline lib/servers_testnet.json]: And also you don't need this comma at the end

**Comment by marceloneil** [comment]
It seems to work great other than some slight syntax issues

**Comment by danny91** [comment]
My bad fixed them now, thanks for noticing.

**Comment by danny91** [comment]
Just checking if this has been forgotten about?

**Comment by marceloneil** [comment]
I haven't had the time to test it out, sorry. But you haven't been forgotten!

**Pull request merged**